src/test/test-execute.c

```diff
@@ -1339,7 +1151,6 @@ static void run_tests(RuntimeScope scope, char **patterns) {
                 entry(test_exec_ignoresigpipe),
                 entry(test_exec_inaccessiblepaths),
                 entry(test_exec_ioschedulingclass),
-                entry(test_exec_mount_apivfs),
```

Comment:
Would it be possible to just skip this test case under sanitizers? I quite like having this as part of the unit test run, rather than the integration test run, as it's much faster feedback

Reply:
A unit test really shouldn't be doing what this test is doing, I think this just fits much better as an integration test. I'd much rather have us focus on speeding up the integration tests rather than trying to fit stuff that should be an integration test in a unit test.

Reply:
An integration test running in a VM will never be able to run in a handful of seconds though, and it also won't be able to run at package build time. Of course I'm not asking to make it work under sanitizers, that would be way too involved, and having the same test case also in the integration test is nice. But I'd really like to keep this, and just have it skipped if sanitizers are enabled, that should be simple enough?